OpenAI-DotNet/OpenAIClientSettings.cs

```diff
@@ -96,13 +98,16 @@ public OpenAIClientSettings(string resourceName, string deploymentId, string api
             BaseRequest = $"/openai/deployments/{DeploymentId}/";
             BaseRequestUrlFormat = $"https://{ResourceName}.{AzureOpenAIDomain}{BaseRequest}{{0}}?api-version={ApiVersion}";
             UseOAuthAuthentication = useActiveDirectoryAuthentication;
+            PooledConnectionLifetime = pooledConnectionLifetime ?? TimeSpan.FromMinutes(15);
         }
 
         public string ResourceName { get; }
 
         public string ApiVersion { get; }
 
         public string DeploymentId { get; }
+        
+        public TimeSpan PooledConnectionLifetime { get; }
```

Comment:
Does this need to be public?

Reply:
I was following the pattern of the properties above it. Seeing as how it's a readonly property I don't think much harm would be caused by it being public. I can change if you would prefer `internal` or something else.